lib/slipstream/serializer/phoenix_socket_v2_serializer.ex

```diff
@@ -103,9 +103,11 @@ defmodule Slipstream.Serializer.PhoenixSocketV2Serializer do
   defp decode_binary!(<<
          @push::size(8),
```

Comment:
I think the current implementation for `decode_binary!/1` is correct: this clause is decoding binary pushes and the clause below is for replies. I would need to check the Phoenix code again but I think only the reply binary has the ref_size and ref parts.